src/plugins/utcTimeSystem/UTCTimeFormat.js

```diff
@@ -20,75 +20,62 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
-define([
-    'moment'
-], function (
-    moment
-) {
-    const DATE_FORMAT = "YYYY-MM-DD HH:mm:ss.SSS";
-    const DATE_FORMATS = [
-        DATE_FORMAT,
-        DATE_FORMAT + "Z",
-        "YYYY-MM-DD HH:mm:ss",
-        "YYYY-MM-DD HH:mm",
-        "YYYY-MM-DD"
-    ];
+import moment from 'moment';
 
-    /**
-     * @typedef Scale
-     * @property {number} min the minimum scale value, in ms
-     * @property {number} max the maximum scale value, in ms
-     */
+const DATE_FORMAT = "YYYY-MM-DD HH:mm:ss.SSS";
+const DATE_FORMATS = [
+    DATE_FORMAT,
+    DATE_FORMAT + "Z",
+    "YYYY-MM-DD HH:mm:ss",
+    "YYYY-MM-DD HH:mm",
+    "YYYY-MM-DD"
+];
 
-    /**
-     * Formatter for UTC timestamps. Interprets numeric values as
-     * milliseconds since the start of 1970.
-     *
-     * @implements {Format}
-     * @constructor
-     * @memberof platform/commonUI/formats
-     */
-    function UTCTimeFormat() {
-        this.key = "utc";
-    }
+/**
+ * @typedef Scale
+ * @property {number} min the minimum scale value, in ms
+ * @property {number} max the maximum scale value, in ms
+ */
 
-    /**
-     * @param {string} formatString
-     * @returns the value of formatString if the value is a string type and exists in the DATE_FORMATS array; otherwise the DATE_FORMAT value.
-     */
-    function validateFormatString(formatString) {
-        return typeof formatString === 'string' && DATE_FORMATS.includes(formatString) ? formatString : DATE_FORMAT;
+/**
+ * Formatter for UTC timestamps. Interprets numeric values as
+ * milliseconds since the start of 1970.
+ *
+ * @implements {Format}
+ * @constructor
+ * @memberof platform/commonUI/formats
+ */
+class UTCTimeFormat {
+    constructor() {
+        this.key = "utc";
     }
 
     /**
      * @param {number} value The value to format.
-     * @param {string} formatString The string format to format. Default "YYYY-MM-DD HH:mm:ss.SSS" + "Z"
-     * @returns {string} the formatted date(s) according to the proper parameter of formatString or the default value of "YYYY-MM-DD HH:mm:ss.SSS" + "Z".
-     * If multiple values were requested, then an array of
+     * @returns {string} the formatted date(s). If multiple values were requested, then an array of
      * formatted values will be returned. Where a value could not be formatted, `undefined` will be returned at its position
      * in the array.
      */
-    UTCTimeFormat.prototype.format = function (value, formatString) {
+    format(value) {
```

Comment:
Does this mean we're not allowing user's to specify the `formatString` anymore? Looks like a breaking change here.

Reply:
Stand by, I think something odd is going on here. It looks like another PR was merged in the mean time that also migrates the UTCTimeFormat to a plugin :)

Reply:
Fixed